app/views/announcements/index.html.erb

```diff
@@ -0,0 +1,46 @@
+<header>
+  HEADER
+</header>
+
+<body>
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-md-3 sidebar">
+        <h1>Announcements</h1>
+
+          <% @announcements.each do |a| %>
+            <div class="post">
+              <div class="title"><%= a.subject %></div>
+              <div class="date">Posted on <%= format_time(a.created_at) %></div>
+              <div class="content">
+                <%= truncate(a.details, length: AnnouncementsHelper::DETAILS_LENGTH) %>
```

Comment:
I think you can just use `DETAILS_LENGTH` here, have you tried?

Reply:
Yups, it doesn't work if I just put `DETAILS_LENGTH`.